xbmc/video/VideoDatabase.cpp

```diff
@@ -4566,6 +4589,13 @@ void CVideoDatabase::UpdateTables(int iVersion)
       }
     }
   }
+  if (iVersion < 94)
```

Comment:
No that's not 100% correct. If you change a `CREATE VIEW` statement there will be no update instructions but you still need to bump the version so that the views are dropped and re-created during the update procedure.

It's for alpha users but most importantly for developers which run through all kinds of versions of the source code and therefore also the database layout. Furthermore these update procedures are also processed one by one when updating from one stable version to the next up to the version specified in `GetSchemaVersion()`.